sql/core/src/test/scala/org/apache/spark/sql/connector/KeyGroupedPartitioningSuite.scala

```diff
@@ -1558,7 +1558,7 @@ class KeyGroupedPartitioningSuite extends DistributionAndOrderingSuiteBase {
     }
   }
 
-  test("SPARK-47094: Support compatible buckets with common divisor") {
+  test("SPARK-47094: SPJ:Support compatible buckets with common divisor") {
```

Comment:
Nit: space before Support ?  Maybe SPJ is redundant here actually

Reply:
All the other ones in SPJ has this `SPJ:`

Reply:
I see, its fine then, maybe just put the space